src/platforms/node/common/profiling/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Profiling
+description: "Learn more about how to configure our Profiling integration and start profiling your code."
+---
+
+By default, Sentry error events will not get trace context unless you configure the scope with the transaction, as illustrated in the example below.
+
+<Note>
+
+Node.js profiling is currently in alpha, which means that it's still in progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [profiling@sentry.io](mailto:profiling@sentry.io). If you’re adopting Profiling in a high-throughput environment, we recommend testing prior to deployment to ensure that your service’s performance characteristics maintain expectations.
+
+</Note>
+
+```javascript
+const Sentry = require("@sentry/node");
+// Note: You MUST import @sentry/tracing package before @sentry/profiling-node
+require("@sentry/tracing");
+const {ProfilingIntegration} = require("@sentry/profiling-node");
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [
+    // Add our Profilling integration
+    new ProfilingIntegration(),
+  ],
+  tracesSampleRate: 1.0,
+  // Set sampling rate for profiling
+  profilesSampleRate: 1.0,
+});
+
+const transaction = Sentry.startTransaction({
+  op: "transaction",
+  name: "My Transaction",
+});
+
+// Any code executed between startTransaction and transaction.finish
+// will now be automatically profiled.
+
+transaction.finish();
+```
+
+
+## How Does It Work?
+
+Under the hood, the sentry profiler leverages V8's [CpuProfiler](https://v8docs.nodesource.com/node-18.2/d2/d34/classv8_1_1_cpu_profiler.html) to collect stack samples. This means that sentry/profiling-node is written as a [native add-on](https://nodejs.org/docs/latest-v18.x/api/addons.html) for Node and won't run in environments like Deno or Bun. Profiling enhances tracing by providing profiles for individual transactions. This allows you to look at higher level performance information like transaction and span durations before diving deeper and looking at profiles.
```

Comment:
Should "Sentry" in the first sentence be capitalized? I get why it isn't in the second sentence when it's part of the a larger term. I changed "leverages" to "uses" to try to simplify the language as much as possible.

```suggestion
Under the hood, the Sentry profiler uses V8's [CpuProfiler](https://v8docs.nodesource.com/node-18.2/d2/d34/classv8_1_1_cpu_profiler.html) to collect stack samples. This means that sentry/profiling-node is written as a [native add-on](https://nodejs.org/docs/latest-v18.x/api/addons.html) for Node and won't run in environments like Deno or Bun. Profiling enhances tracing by providing profiles for individual transactions. This allows you to look at higher level performance information like transaction and span durations before diving deeper and looking at profiles.
```